### PR TITLE
Remove product content guide

### PR DESCRIPTION
**Pull request by mmartinsoliv**
[JIRA Issue](https://gympass.atlassian.net/browse/DS-351)

## Description 📄

Remove product content guide

## Platforms 📲

- [ ] Web
- [ ] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [ ] Snapshot Test

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

[Upload your screenshots here]

| Before | After |
| ------ | ----- |
|image|image|

### CODE REVIEW

File: packages/doc/src/components/Layout/Layout.jsx [hunk 1]
```diff
@@ -131,9 +131,7 @@ const Layout = ({
       <MainWrapper>
         <Grid>
           <Header showMenu={showMenu} toggleMenu={toggleMenu}>
-            <HeaderLink to="/guidelines/product-content/introduction">
-              Guidelines
-            </HeaderLink>
+            <HeaderLink to="/guidelines/tokens/borders">Guidelines</HeaderLink>
```
Comment (ericcleao) [inline packages/doc/src/components/Layout/Layout.jsx:134]: Why click on the `Guidelines` should lead the user to `/guidelines/tokens/borders` and not `/guidelines`?

**Review by leonardokl — APPROVED** [review]

File: packages/doc/src/pages/index.jsx [hunk 2]
```diff
@@ -68,7 +68,7 @@ const boxes = [
     title: 'guidelines',
     description:
       'Design principles, practical patterns and high quality design resources to help people create their product prototypes beautifully and efficiently.',
-    href: '/guidelines/product-content/introduction',
+    href: '/guidelines/tokens/borders',
```
Comment (ericcleao) [inline packages/doc/src/pages/index.jsx:71]: Same question from the `Layout` file.

File: packages/doc/src/components/Layout/Layout.jsx (hunk 1 above)
Comment (mmartinsoliv) [reply packages/doc/src/components/Layout/Layout.jsx:134]: If you keep only on the `/guidelines` route, you will be redirected to an empty screen.

Comment (mmartinsoliv) [reply packages/doc/src/components/Layout/Layout.jsx:134]: Example:
image

**Review by ericcleao — APPROVED** [review]

**Pull request merged**